docs/faq/README.md

```diff
@@ -87,6 +64,42 @@ Regardless of storage option used, IPAM controller expects a directory volume mo
 
 In the mount directory, rename or remove a file named `cis_ipam.sqlite3`.
 
+## Troubleshooting
+
+### How to troubleshoot FIC pod logs ?
+
+`kubectl logs deploy/<name-of-ipam-deployment> -n kube-system -f`
+
+### Error - `Unable to Update IPAM: kube-system/***  Error: ipams.fic.f5.com "***" not found`
+
+* Verify cluster role permissions for ipam resource.
+
+  - `kubectl describe clusterrole <name> -n kube-system | grep ipam`
```

Comment:
@sravyap135  - What would be the expected example output here ? please share the expected output so that users  can verify the same.